dune-project

```diff
@@ -21,8 +21,9 @@
   (dune (>= 2.3))
   (emile (>= 1.0))
   (mrmime (>= 0.3.1))
```

Comment:
https://github.com/dinosaure/opam-repository/commit/7b29b54bc7b806246eb2ff8b8f5b766f427b68de (at the bottom of the commit body) suggests we upgrade `mrmime` to `0.3.2` too

Reply:
Our dependency is really to `0.3.1` or more recent. I've updated the lock file to have `0.3.2`.

Reply:
Thanks!